caffe2/utils/math_gpu.cu

```diff
@@ -204,6 +204,98 @@ void Gemm<float16, CUDAContext>(
   }
 }
 
+#if CUDA_VERSION >= 9000
+
+// No change, but required. Defer to default CUDA engine
+template <>
+void Gemm<float, CUDAContext, TensorCoreEngine>(
+    const CBLAS_TRANSPOSE TransA,
+    const CBLAS_TRANSPOSE TransB,
+    const int M,
+    const int N,
+    const int K,
+    const float alpha,
+    const float* A,
+    const float* B,
+    const float beta,
+    float* C,
+    CUDAContext* context,
+    TensorProto::DataType math_type) {
+  return Gemm<float,CUDAContext>(TransA,
```

Comment:
Yes. Unless I'm misunderstanding this will use the default engine, which should be cuBLAS for CUDAContext.